### PR TITLE
Adding metrics for UpdateCertificateRetrievalErrors and UpdateMissingCertificates

### PR DESCRIPTION
**Pull request by anispate**
Figuring out bug as part of [OSD-14653](https://issues.redhat.com/browse/OSD-14653)

### CODE REVIEW

File: pkg/localmetrics/localmetrics.go [hunk 1]
```diff
@@ -193,3 +202,19 @@ func IncrementLetsEncryptMaintenanceErrorCount() {
 func IncrementDnsErrorCount() {
 	MetricDnsErrorCount.Inc()
 }
+
+// register missingCertificates and certificateRetrievalErrors metrics in the prometheus
+func init() {
+	metrics.Registry.MustRegister(missingCertificates)
+	metrics.Registry.MustRegister(certificateRetrievalErrors)
+}
```
Comment (iamkirkbater) [inline pkg/localmetrics/localmetrics.go:210]: Is there a reason we're initializing these two metrics here? How are the other metrics initialized? Should we try to initialize them the same way as the others?

**Review by iamkirkbater — CHANGES_REQUESTED** [review]

File: pkg/localmetrics/localmetrics.go (hunk 1 above)
Comment (anispate) [reply pkg/localmetrics/localmetrics.go:210]: @iamkirkbater  i wasn't able to find anyway they are initializing these metrics the current way therefore, i initalized these new metrics the new way. However, ill be reverting back the new metrics changes once I find the root cause on why current metrics are not getting all the certs inc hibernating, error and LS clusters.

**Comment by iamkirkbater** [comment]
/hold

Should be safe to unhold on Monday, but I'd like to not deploy this so late on a Friday.

/lgtm
/approve

**Pull request merged**